website/www/site/content/en/contribute/committer-guide.md

```diff
@@ -101,8 +101,9 @@ squash, split, etc, the commits, so that the history is most useful:
 * Favor commits that do just one thing. The commit is the smallest unit of easy
 rollback; it is easy to roll back many commits, or a whole pull request, but
 harder to roll back part of a commit.
-* Commit messages should tag JIRAs and be otherwise descriptive.
-It should later not be necessary to find a merge or first PR commit to find out what caused a change.
+* Commit messages should be descriptive. It should later not be necessary to find a
```

Comment:
should commit messages also reference the issue number?

Reply:
Good idea - updated to this